temfpy/uncertainty_quantification.py

```diff
@@ -56,18 +56,28 @@ def eoq_model(x, r=0.1):
     and the setup costs of an order `x[2]`. The annual interest rate `r` is treated as an
     additional parameter.
 
+    .. math::
```

Comment:
Can we use the same notation as in the docstring above `x[2]`for example instead of x_2?